src/Core/Tools/SendFeatures/Services/Interfaces/ISendStorageService.cs

```diff
@@ -3,14 +3,61 @@
 
 namespace Bit.Core.Tools.Services;
 
+/// <summary>
+/// Send File Storage Service is responsible for uploading, deleting, and validating files
+/// whether they are in local storage or in cloud storage.
+/// </summary>
 public interface ISendFileStorageService
 {
     FileUploadType FileUploadType { get; }
+    /// <summary>
+    ///  Uploads a new file to the storage.
+    /// </summary>
+    /// <param name="stream"></param>
+    /// <param name="send"></param>
+    /// <param name="fileId"></param>
+    /// <returns></returns>
```

Comment:
⛏️ The parameters and return value should be documented in addition to the summary.